.github/workflows/ci.yml

```
@@ -21,3 +21,23 @@ jobs:
           coverage: xdebug
       - run: composer install
       - run: vendor/bin/phpunit --coverage-text
+
+  PHPStan:
+    name: PHPStan
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php: [ 8.1 ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
```

nit: Remove names for consistency with other actions?

```suggestion
      - uses: actions/checkout@v3
      - uses: shivammathur/setup-php@v2
```

---

To be honest I'd rather we add names to the other steps for consistency x)